README.md

```diff
@@ -169,53 +180,54 @@ vertex-centric index is defined in an object with the following format:
+  - columns: may contain any number of columns(edge property keys)
+    - names: any string 
+    - dataType: String, Long, Integer, and Date. 
+    - composit: true or false
+    - mixedIndex: any String (default should be "search") 
+- relations: Defines how the named edge relations can happen and number of occurrences.
+  - left: the outV of an edge
+  - right: the inV of an edge
+  - supernode: nodes have many edges
+    - vertices: n where n is the first n nodes of left
+    - edges: additional edges are added to each of the n nodes
+    
 ### Import CSV file to JanusGraph
-    JanusGraphBatchImporter is a tool that imports bulk data into JanusGraph using multiple CSV files for the data and json config files to define the schema and mapping between schema and data. It uses multiple workers in order to achieve full speed of import of the data.
-    JanusGraphBatchImporter depends on JanusGraph libraries and also commons-csv-1.4.jar
+JanusGraphBatchImporter is a tool that imports bulk data into JanusGraph using multiple CSV files for the data and json config files to define the schema and mapping between schema and data. It uses multiple workers in order to achieve full speed of import of the data.
 
-    In order to use under Linux:
+JanusGraphBatchImporter depends on JanusGraph libraries and also commons-csv-1.4.jar
 
-    Edit conf/batch_import.properties file to configure the importer
+In order to use under Linux:
+- Edit conf/batch_import.properties file to configure the importer
+- Usage:  
+  `run.sh import <janusgraph-config-file> <data-files-directory> <schema.json> <data-mapping.json>`
+  - `<janusgraph-config-file>`: Properties file with the JanuGraph configuration
+  - `<data-files-directory>:` Relative directory where the data files are located
+  - `<schema.json>:` JSON file defining the schema of the graph
+  - `<data-mapping.json>`: mapping file defining the relationship between the CSV/s fields and the graph
 
-    Usage:
-      run.sh import <janusgraph-config-file> <data-files-directory> <schema.json> <data-mapping.json>
+- Example:
+ `./run.sh import /root/janusgraph-v0.1.1/conf/janusgraph-cassandra-es.properties /tmp /tmp/schema.json /tmp/datamapper.json`
 
-      <janusgraph-config-file>: Properties file with the JanuGraph configuration
-      <data-files-directory>: Relative directory where the data files are located
-      <schema.json>: JSON file defining the schema of the graph
-      <data-mapping.json>: mapping file defining the relationship between the CSV/s fields and the graph
-    Example:
-    ./run.sh import /root/janusgraph-v0.1.1/conf/janusgraph-cassandra-es.properties /tmp /tmp/schema.json /tmp/datamapper.json
-    Using different JanusGraph lib
-    export JANUSGRAPH_HOME=/path-to-your-lib; ./run.sh gencsv csv-conf/tiny_config.json /tmp
+- Using different JanusGraph lib
+    `export JANUSGRAPH_HOME=/path-to-your-lib; ./run.sh gencsv csv-conf/tiny_config.json /tmp`
```

Comment:
My bad. This line should be: 
`export JANUSGRAPH_HOME=/path-to-your-lib;  ./run.sh import /root/janusgraph-v0.1.1/conf/janusgraph-cassandra-es.properties /tmp /tmp/schema.json /tmp/datamapper.json`